src/OpenSage.Game/Data/Apt/ImageMap.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenSage.Mathematics;
+
+namespace OpenSage.Data.Apt
+{
+
+    public interface ImageAssignment
```

Comment:
Interfaces should be prefixed with "I", so please could this be `IImageAssignment`.

Reply:
Yup